.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,26 @@
+name: e2e Test
+
+on: [push, pull_request]
+
+jobs:
+  e2e-tests:
+    strategy:
+      matrix:
+        kong_image:
+        - 'kong:2.7.0'
```

Comment:
Let's do all minor releases until 1.x and see what breaks.

Reply:
You will need to pass this as an "env var" and then read that in the setup-kong.sh script. See https://github.com/Kong/go-kong/blob/main/.github/workflows/integration-test.yaml#L16